client/src/partials/inventory/list/modals/actions.modal.js

```diff
@@ -0,0 +1,106 @@
+angular.module('bhima.controllers')
+.controller('InventoryListActionsModalController', InventoryListActionsModalController);
+
+InventoryListActionsModalController.$inject = [
+  'AccountService', 'InventoryService', 'InventoryGroupService',
+  'InventoryTypeService', 'InventoryUnitService', 'NotifyService',
+  '$uibModalInstance', 'Store', 'data'
+];
+
+function InventoryListActionsModalController(Account, Inventory, InventoryGroup, InventoryType, InventoryUnit, Notify, Instance, Store, Data) {
+  var vm = this, session = vm.session = {};
```

Comment:
Need for an object which contains necessary data to send to the server, VM is more global and contains much more data